src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -77,15 +78,15 @@ class SQLPTRepresentationTest : public SQLPTRepresentation,
   static SQLPTRepresentation* reps;
   static const std::string kDatabaseName;
   //Gtest can show message that this object doesn't destroyed
-  static utils::SharedPtr<policy_handler_test::MockPolicySettings> policy_settings_;
+  static std::auto_ptr<policy_handler_test::MockPolicySettings> policy_settings_;
 
   static void SetUpTestCase() {
     const std::string kAppStorageFolder = "storage1";
     reps = new SQLPTRepresentation;
     dbms = new DBMS(kDatabaseName);
-    policy_settings_ =
-        utils::MakeShared<policy_handler_test::MockPolicySettings>();
-    ON_CALL(*policy_settings_,
+    policy_settings_ = std::auto_ptr<policy_handler_test::MockPolicySettings>(
+          new policy_handler_test::MockPolicySettings());
+    ON_CALL(*policy_settings_.get(),
```

Comment:
@Kozoriz Why do you use here 2 operations `get()` and `*`? For `std::auto_ptr` there is defined `operator *` which will give you access to managed object.